app/jobs/onceoff/voting_ensure_consistency.rb

```diff
@@ -35,6 +35,16 @@ def execute_onceoff(args)
               dvv1.archive = dvv2.archive
       SQL
 
+      # delete votes associated with no user
```

Comment:
Wouldn't making this a new, separate post_migration be a better move? (see the discussion about replacing once-offs with post_migrations)

Reply:
Yes, that was my first intention too. Technically, removing the plugin, deleting some users and then reinstalling the plugin will leave the database in an inconsistent state again.

Reply:
I suspect that's the case with pretty much every plugin that has models? We never do cleanup on plugin uninstallation. Or before installation.

Maybe that's something we should look into. Or decide ("management accepts the risk" 😉) that db inconsistency is something you should expect when removing and re-adding plugins like that. 🤷

Reply:
I think it's fine for now until we have a better story about plugin uninstallation / cleanup.